### ISSUE TITLE
po has always been 0.16.1 not the latest version

### ISSUE THREAD

**Issue by Liyongcong** [bizz84/SwiftyStoreKit#717]
## Bug Report

**To Reproduce**
1. Step one...

**Expected behavior**

## Platform Information
 - OS: [e.g. iOS 13.4, watchOS 6.2.1, tvOS 9.2.3, macOS 10.14.3, Catalyst 13.0]
 - Purchase Type: [e.g. consumable, non-consumable, auto-renewable subscription, non-renewing subscription, discount]
 - Environment: [e.g. sandbox, app review, production]
 - SwiftyStoreKit version: [e.g. 0.16]

## Additional context

**Potentially Related Issues**
 - Issue #___

**Screenshots**

**Comment by Giggs079** [comment]
I had the same issue with you when install SwiftyStoreKit with CocoaPods, I cannot install 0.16.2 and later versions, only 0.16.1
But I found a solution, if you uninstall SwiftyStoreKit from CocoaPods and then install it again using Swift Package Manager you can install the last version 0.16.4

**Comment by Liyongcong** [comment]
CocoaPods is mainly used in the project. Can Swift Package Manager and CocoaPods be used at the same time

**Comment by Liyongcong** [comment]
> I had the same issue with you when install SwiftyStoreKit with CocoaPods, I cannot install 0.16.2 and later versions, only 0.16.1 But I found a solution, if you uninstall SwiftyStoreKit from CocoaPods and then install it again using Swift Package Manager you can install the last version 0.16.4

CocoaPods is mainly used in the project. Can Swift Package Manager and CocoaPods be used at the same time，And I encountered a problem, when evoking the Apple Pay page, the evoking failed, the configuration of things are no problem, have you ever encountered

**Comment by Giggs079** [comment]
Yes you can use Swift Package Manager and CocoaPods for dependencies at the same time.
I have some dependencies with CocoaPods because these pods have not yet a Swift Package Manager version so I got to keep them in CocoaPods and I have my other dependencies added in Swift Package Manager so I use both and I have no problem. Just to be sure I haven't the same dependency in both, you should remove it from CocoaPods before add it in Swift Package Manager.

For your Apple Pay issue, I don't know about it, we don't use it in our app. I didn't know SwiftyStoreKit handle Apple Pay too
